session_security/urls.py

```diff
@@ -21,10 +21,10 @@
 
 from .views import PingView
 
-urlpatterns = patterns('',
+urlpatterns = [
     url(
         'ping/$',
         PingView.as_view(),
         name='session_security_ping',
-    ),
-)
+    )
+]
```

Comment:
I think this is going to break with django < 1.8, which doesn't matter to me, but in that case those versions should be removed from travis.

Good job bro !!